content/en/events/2023/kcsna/faq.md

```diff
@@ -0,0 +1,119 @@
+- [Why do I need to be a Kubernetes Org member to attend in-person?](#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person)
+- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
+- [I have an idea for a session](#i-have-an-idea-for-a-session)
+- [My SIG wants to meet at the Summit](#my-sig-wants-to-meet-at-the-summit)
+- [Will there be a doc sprint?](#will-there-be-a-doc-sprint)
+- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [Can I bring a guest to the Social?](#can-i-bring-a-guest-to-the-social)
+- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)
+
+### When is the summit taking place?
+
+November 6th, the first day of [KubeCon North America].
+
+[KubeCon North America]: https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/
+
+### Where is the summit located?
+
+The in-person location at the
+<a href="https://www.hyatt.com/en-US/hotel/illinois/hyatt-regency-mccormick-place/chimc" rel="noopener noreferrer" target="_blank">Hyatt Regency McCormick Place</a>,
+See the [Location & Venue] section for more information.
+
+[Location & Venue]: /events/2023/kcsna/location/
+
+### Will I be able to attend remotely?
+
+No, based on our experience with previous Contributor Summits, we do not expect to be able
+to support remote access for this event.
+
+### What are the Health and Safety protocols?
+
+As a co-located event, the summit is required to adhere to KubeCon’s
+<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/attend/health-and-safety/" rel="noopener noreferrer" target="_blank">
+Health & Safety Guidelines</a>. There may be additional restrictions that
+develop closer to the event, and we will update the event accordingly to follow
+venue, municipality and CDC guidelines at a minimum.
+
+### Why do I need to be a Kubernetes Org member to attend in-person?
+
+We welcome everyone to start contributing to the project, but with 1600+
+Kubernetes Org members and limited space and budget requirements we've
+made the decision to limit attendance to those that are already actively
+engaged with the project.
+
+If you are not yet a member, but active within the project, please consider
+joining! The process of joining is straight forward. See our [org membership]
+docs for more information. If your application to join is pending, please
+contact us at community@kubernetes.io about registering.
+
+Kubernetes Orgs:
+
+<ul>
+<li><a href="https://github.com/kubernetes" rel="noopener noreferrer" target="_blank">kubernetes</a></li>
+<li><a href="https://github.com/kubernetes-client" rel="noopener noreferrer" target="_blank">kubernetes-client</a></li>
+<li><a href="https://github.com/kubernetes-csi rel="noopener noreferrer" target="_blank">kubernetes-csi</a></li>
+<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
+</ul>
```

Comment:
Not necessarily something to add to the list, but an FYI for reg we should also accept etcd as they're progressing towards being a part of k8s.

Reply:
Totally forgot, thank you!! Will also update the registration playbook to include etcd folks check, will do it later this/next week.

Reply:
Suggested text:

This year, Etcd contributors with 50 or more contributions in the last year are also invited to attend, in anticipation of [Etcd becoming a Kubernetes SIG](https://github.com/kubernetes/community/pull/7372).

Reply:
I think we can just flag org membership as the general baseline, but if they aren't an org member there yet and have contributed a fair amount to still reach out to us.

Reply:
The org membership program for Etcd is brand-new, and most people aren't signed up for it yet.   Including some of the maintainers.  I can provide a list of < 50contributions.

Reply:
@jberkus Will it make sense to include a devstats link so folks can check eligibility before registering? (i.e. If I made some etcd contributions last summer, am I invited or not?). 
Or I can put something like "Etcd contributors are [now](https://github.com/kubernetes/community/pull/7372) eligible to be part of Kubernetes org and more than welcome at the event"?

Reply:
Gah, this requires more explanation/details.  It's going to be easier to just send them to the Etcd Devs directly.  Maybe just:

This year, Etcd project Members and regular contributors are eligible to attend the Contributor Summit.  More information on this will be sent to the Etcd Developer mailing list.